src/Microsoft.DotNet.Build.Tasks/Targets/publishtest.targets

```diff
@@ -36,8 +31,6 @@
   </ItemGroup>
 
   <Target Name="CopyTestToTestDirectory"
-    Inputs="@(PackagesConfig);$(TestRuntimePackageConfig);@(ReferenceCopyLocalPaths);@(Content);@(IntermediateAssembly);@(_DebugSymbolsIntermediatePath)"
```

Comment:
The problem with using Outputs on this task - and the reason I ultimately gave up on making this target independently incremental and invoked it from RunTestsForProject instead - is that MSBuild requires Outputs to be statically identified; it will not evaluate wildcards. The reasons are explained [here](http://blogs.msdn.com/b/msbuild/archive/2006/02/22/537150.aspx) (the reasons for not using wildcards are sound, but I would have preferred the MSBuild team to allow the developer to make that decision rather than enforcing it in the tool).

Outputs for the copy cannot easily be statically determined since they include packages references. The package resolution could be moved into its own task that runs prior to the copy, which would allow the outputs for the copy to be statically determined; and in fact I have a local commit that does just that. However, the result was increased complexity in the build files, as well as the running build (timestamps on the entire test directory now have to be compared), for what I felt was little value; executing the copy from inside RunTestsForProject felt like a cleaner solution.

If you would like, I can push my local commit to a branch so you can see what it would look like.